test/controllers/api/rhsm/candlepin_dynflow_proxy_controller_integration_test.rb

```diff
@@ -14,12 +14,14 @@ class Api::Rhsm::CandlepinDynflowProxyControllerIntegrationTest < ActionDispatch
     end
 
     test 'params are not parsed in the controller' do
-      JSON.expects(:parse).never
       Api::Rhsm::CandlepinDynflowProxyController.any_instance.expects(:async_task).returns(nil)
       Resources::Candlepin::Consumer.expects(:get).returns({})
+      packages = [{"vendor" => "CentOS", "name" => "python-six", "epoch" => 0, "version" => "1.9.0", "release" => "2.el7", "arch" => "noarch"}]
```

Comment:
This is inconsequential to the test, but I wanted to have a more accurate representation of what might actually be sent through the API